start/src/main/java/org/apache/accumulo/start/Main.java

```diff
@@ -45,13 +44,11 @@ public static void main(final String[] args) throws Exception {
     // Preload classes that cause a deadlock between the ServiceLoader and the DFSClient when
     // using the VFSClassLoader with jars in HDFS.
     ClassLoader loader = getClassLoader();
```

Comment:
Do we need the `getClassLoader()` method anymore? Can we just call `ClassLoaderUtil.getClassLoader(null);` ?

Reply:
I actually tried to do that but the issue is ClassLoaderUtil isn't available to use here. The main class is part of the `start` module and ClassLoaderUtil lives inside the `core` module. Core has a dependency on start so we can't use the class.